### DISCUSSION TITLE
MonitorControl causes oblique bars to appear on external Samsung monitor

### DISCUSSION THREAD

**Discussion by syxanash** [MonitorControl/MonitorControl#1475] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

Hi I've noticed that after installing MonitorControl on my Macbook Air M2 2022 I'm noticing some oblique bars appearing all around my external monitor (see attached pictures). The external monitor is a SAMSUNG LS24C366EAUXXU. When I adjust the brightness, these might go away randomly. The external monitor is brand new and the same issue does not appear on another Macbook where I did not install MonitorControl.

![IMG_5777](https://github.com/MonitorControl/MonitorControl/assets/1325491/8c091e52-7d86-45a4-af62-cb394d9702b6)
![IMG_5778](https://github.com/MonitorControl/MonitorControl/assets/1325491/3ab6e9bb-c7fd-46ab-9d0c-d78c51df2be1)

Last picture is a normal scenario where the issue does not happen while tweaking the brightness with MonitorControl:

![IMG_5779](https://github.com/MonitorControl/MonitorControl/assets/1325491/b5648aca-24dc-42f0-9741-ff90a6605d52)

### Steps to reproduce

1. Randomly tweak brightness on MonitorControl
2. Oblique bars appear on certain brightness level

### Expected behavior

There shouldn't be any oblique bars but the brightness should just be adjusted.

### Anything else?

These are my configuration params:

![Screenshot 2023-11-25 at 17 44 17](https://github.com/MonitorControl/MonitorControl/assets/1325491/bb1a2235-0d2f-48e8-8243-b56ae42126e4)
![Screenshot 2023-11-25 at 17 44 09](https://github.com/MonitorControl/MonitorControl/assets/1325491/ed1eb132-5e51-486c-b5ce-da57dde807ed)

### Environment Information (please complete the following information)

```markdown
- macOS version: Sonoma 14.1.1 (23B81)
- Mac model: Macbook Air M2 2022
- MonitorControl version: 4.2.0 build 7048
- Monitor(s): SAMSUNG LS24C366EAUXXU
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment]
This might be some kind of panel issue. I notice that DDC is disabled so the app is simply changing colors and the display might not reproduce some shades properly.
